stats/stats/justfile

```diff
@@ -17,4 +17,4 @@ generate-entities database_url="postgres://postgres:admin@localhost:5432/stats":
     DATABASE_URL={{database_url}} sea-orm-cli generate entity --lib -o entity/src
     cargo fmt --all -- --config imports_granularity=Crate
 
-restart-generate-entities: restart-postgres migrate-up generate-entities stop-postgres
+# restart-generate-entities: restart-postgres migrate-up generate-entities stop-postgres
```

Comment:
why?

Reply:
there is no "restart postgres" in this file, I've moved it into stats root dir